source/options.html

```diff
@@ -45,7 +40,7 @@ <h2 class="only-firefox">Options</h2>
 	<hr>
 
 	<p>
-		Want more features? Check out our <a href="https://github.com/sindresorhus/refined-github#related" target="_blank">related GitHub extensions</a>
+		Want more? Check out our <a href="https://github.com/sindresorhus/refined-github#related" target="_blank">related GitHub extensions</a> or <a href="https://github.com/sindresorhus/refined-github/blob/master/contributing.md" target="_blank">contribute</a>!
```

Comment:
```suggestion
		Want more? Check some <a href="https://github.com/sindresorhus/refined-github#related" target="_blank">related GitHub extensions</a> or <a href="https://github.com/sindresorhus/refined-github/blob/master/contributing.md" target="_blank">contribute</a>!
```

It's not really just "ours".

Reply:
I think it should still be `check out some` and not just `check`